conf/machine/imx8mp-evk.conf

```diff
@@ -4,13 +4,13 @@
 #@DESCRIPTION: Machine configuration for NXP i.MX 8M Plus Evaluation Kit
 #@MAINTAINER: Jun Zhu <junzhu@nxp.com>
 
-MACHINEOVERRIDES =. "mx8:mx8m:mx8mp:"
+MACHINEOVERRIDES =. "boot-container:mx8:mx8m:mx8mp:"
 
 require conf/machine/include/imx-base.inc
 require conf/machine/include/tune-cortexa53.inc
 
-MACHINE_FEATURES += "pci wifi bluetooth optee jailhouse"
-MACHINE_FEATURES_append_use-nxp-bsp = " mrvl8997"
+MACHINE_FEATURES += "pci wifi bluetooth jailhouse"
+MACHINE_FEATURES_append_use-nxp-bsp = " optee mrvl8997"
```

Comment:
There is a comment in imx8m{m,n}-evk.conf files that I think is valid here too?
"
    # OP-TEE is also applicable to NXP BSP, mainline BSP seems not to have
    # a full support for it yet.
"